src/guild/roleList.ts

```diff
@@ -11,10 +11,10 @@ export interface RoleList {
 	robot?: string;
 	buildTeam?: string;
 	builder?: string;
-	support?: string;
+	helper?: string;
 	staff?: string;
 	mentor?: string;
-	juniorModerator?: string;
+	superhelper?: string;
```

Comment:
Would make `superhelper` camel case to be consistent so it would be `superHelper`